### PR TITLE
Introduce StructuralAnalysis:TaperedMaterialProfile class

### PR DESCRIPTION
**Pull request by laurynas111**
Introduce a new TaperedMaterialProfile class for Structural Analysis Schema
This class allows defining linearly varying Tapered Curve Members.

### CODE REVIEW

File: Domains/3-DisciplineOther/StructuralAnalysis/StructuralAnalysis.ecschema.xml [hunk 1]
```diff
@@ -833,12 +835,30 @@
...
         <BaseClass>MaterialProfileDefinition</BaseClass>
-        <ECProperty propertyName="Category" typeName="MaterialProfileCategoryType" description="Role of the MaterialProfile." />
-        <ECNavigationProperty propertyName="Material" relationshipName="MaterialProfileRefersToMaterial" direction="Forward" description="Referenced PhysicalMaterial." />
         <ECNavigationProperty propertyName="Profile" relationshipName="MaterialProfileRefersToProfile" direction="Forward" description="Referenced Profile." />
     </ECEntityClass>
 
+    <ECRelationshipClass typeName="MaterialProfileDefinitionRefersToMaterial" strength="referencing" strengthDirection="Forward" modifier="None">
+        <ECCustomAttributes>
+            <Deprecated xmlns="CoreCustomAttributes.1.0.3">
+                <Description>.</Description>
```
[the remark this answers is not in the document]
Comment (laurynas111) [reply Domains/3-DisciplineOther/StructuralAnalysis/StructuralAnalysis.ecschema.xml]: Thanks for pointing that out

File: Domains/3-DisciplineOther/StructuralAnalysis/StructuralAnalysis.ecschema.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 |  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 |  * See LICENSE.md in the project root for license terms and full copyright notice.
 ======================================================================================= -->
-<ECSchema schemaName="StructuralAnalysis" alias="sa" version="01.00.03" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2" displayLabel="Structural Analysis" description="Structural Analysis Schema.">
+<ECSchema schemaName="StructuralAnalysis" alias="sa" version="01.00.02" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2" displayLabel="Structural Analysis" description="Structural Analysis Schema.">
```
Comment (diegoalexdiaz) [inline Domains/3-DisciplineOther/StructuralAnalysis/StructuralAnalysis.ecschema.xml:6]: WIP schemas should always be ahead of the last released version. I'm seeing that the last released version of this schema is 1.0.2, thus this WIP version should be 1.0.3.

File: Domains/3-DisciplineOther/StructuralAnalysis/StructuralAnalysis.ecschema.xml [hunk 3]
```diff
@@ -815,14 +815,16 @@
             <Class class="MaterialProfileType" />
         </Source>
         <Target multiplicity="(1..1)" roleLabel="is referenced by" polymorphic="false">
-            <Class class="MaterialProfile" />
+            <Class class="MaterialProfileDefinition" />
         </Target>
     </ECRelationshipClass>
 
-    <ECEntityClass typeName="MaterialProfileDefinition" modifier="abstract" displayLabel="Material Profile Definition" description="Base class defining an element that pairs Profiles and PhysicalMaterials.">
+    <ECEntityClass typeName="MaterialProfileDefinition" modifier="None" displayLabel="Material Profile Definition" description="Base class defining an element that pairs Profiles and PhysicalMaterials.">
```
Comment (diegoalexdiaz) [inline Domains/3-DisciplineOther/StructuralAnalysis/StructuralAnalysis.ecschema.xml:822]: Was this change from "abstract" to concrete intended? It feels strange looking at the other changes in this PR (i.e. MaterialProfileDefinition continues to be used primarily as a base class).

File: Domains/3-DisciplineOther/StructuralAnalysis/StructuralAnalysis.ecschema.xml (hunk 2 above)
Comment (laurynas111) [reply Domains/3-DisciplineOther/StructuralAnalysis/StructuralAnalysis.ecschema.xml]: Updated this. Thanks.

File: Domains/3-DisciplineOther/StructuralAnalysis/StructuralAnalysis.ecschema.xml (hunk 3 above)
Comment (laurynas111) [reply Domains/3-DisciplineOther/StructuralAnalysis/StructuralAnalysis.ecschema.xml]: Thanks for noticing this.

**Review by karolis-zukauskas — APPROVED** [review]

**Pull request merged**